Adding content on the readme file (#22)
sgibson91/cross-stitch-carpentry: README.md

> # Let's start cross stich

Reviewer: ```suggestion
# How to Cross Stitch
```

This repository contains chapters on cross stitching, primarily written by Dr. Sarah Gibson.

> [_The Turing Way_](https://github.com/alan-turing-institute/the-turing-way) team members wanted to test The Carpentries lesson Template so that they could use it for hosting lessons relevant for The Turing Way project.

Reviewer: ```suggestion
[_The Turing Way_](https://github.com/alan-turing-institute/the-turing-way) team members wanted to test The Carpentries [lesson template](https://github.com/carpentries/lesson-example) so that they could use it for hosting lessons relevant for _The Turing Way_ project.
```

> A lesson on cross stitch like a good example to start with for the following reasons:

Reviewer: ```suggestion
A lesson on cross stitch felt like a good example to start with for the following reasons:
```

> - _The Turing Way_ team (with the great help of Sarah) gives out a starter's package with a how-to manual to their book dash attendees as a thank you gift.

Reviewer: ```suggestion
- _The Turing Way_ team (with the great help of Sarah) gave out a starter packages with a how-to manual to their book dash attendees as a thank you gift
```

> - Sarah , who is a research software engineer, has discovered that as a data scientists this hobby allows her to use her analytical skills in a hobby

Reviewer: ```suggestion
- Sarah , who is a research software engineer, has discovered that cross stitch allows her to gently exercise her analytical skills in a hobby environment
```
